src/views/RootStackScreens/TopicScreen.tsx

```diff
@@ -1,10 +1,12 @@
 import type { StackScreenPropsGeneric } from "@src/@types/navigation";
 import { TopicView } from "../TopicView";
 
-export type TopicScreenProps = undefined;
+export type TopicScreenProps = {
+  tid: string;
+};
```

Comment:
It may be confused to just use `tid` as a variable name. Just name it as `id` (we know it's the unique identify number of the topic), or add your document with comments. Do this for all your exported interface.
```suggestion
export type TopicScreenProps = {
  id: string; // The topic id
};
```